packages/client/modules/teamDashboard/components/TeamDashInsightsTab/TeamInsights.tsx

```diff
@@ -22,23 +32,101 @@ const query = graphql`
 const Insights = (props: Props) => {
   const {queryRef} = props
   const data = usePreloadedQuery<TeamInsightsQuery>(query, queryRef)
-  // TODO: use the query rather than just console logging it
-  console.log('🚀 ~ data:', data)
+  const {viewer} = data
+  const {team} = viewer
+  const {insight, name} = team ?? {}
+
+  const renderMarkdown = (text: string) => {
+    const renderedText = marked(text, {
+      gfm: true,
+      breaks: true
+    }) as string
+    return sanitizeHtml(renderedText, {
+      allowedTags: sanitizeHtml.defaults.allowedTags.concat(['a']),
+      allowedAttributes: {
+        ...sanitizeHtml.defaults.allowedAttributes,
+        a: ['href', 'target', 'rel']
+      },
+      transformTags: {
+        a: (tagName, attribs) => {
+          return {
+            tagName,
+            attribs: {
+              ...attribs,
+              target: '_blank',
+              rel: 'noopener noreferrer'
+            }
+          }
+        }
+      }
+    })
+  }
+
+  const formatDateRange = (start: string, end: string) => {
+    const startDate = dayjs(start)
+    const endDate = dayjs(end)
+
+    if (startDate.year() === endDate.year()) {
+      return `${startDate.format('MMM')} to ${endDate.format('MMM YYYY')}`
+    } else {
+      return `${startDate.format('MMM YYYY')} to ${endDate.format('MMM YYYY')}`
+    }
+  }
+
+  const dateRange = insight
+    ? formatDateRange(insight.startDateTime, insight.endDateTime)
+    : 'Date range not available'
 
   return (
     <div className='mb-8 space-y-6'>
       <p className='mb-6 mt-[20px] text-sm text-slate-900'>
         Only you (as <span className='font-bold'>Team Lead</span>) can see Team Insights. Insights
         are auto-generated.{' '}
-        <a href='#' className='font-semibold text-sky-500 hover:underline'>
+        <a
+          href='#'
+          className='font-semibold text-sky-500 hover:underline'
+          target='_blank'
+          rel='noopener noreferrer'
+        >
           Give us feedback
         </a>
       </p>
       <div className='mx-auto aspect-[1/1.414] w-[640px] max-w-3xl overflow-y-auto rounded-lg bg-white px-[56px] pt-8 shadow-md'>
         <h2 className='mb-4 mt-0 flex items-center pt-0 text-2xl font-semibold leading-9'>
           <AutoAwesomeIcon className='mr-2 h-9 w-9 text-grape-500' />
-          <span>Insights - Aug to Sep 2024</span>
+          <span>Insights - {dateRange}</span>
         </h2>
+        <p className='mb-6 text-sm text-slate-600'>Summarized {insight?.meetingsCount} meetings</p>
```

Comment:
I'll add empty state soon (https://github.com/ParabolInc/parabol/issues/10281) so there's no need to check if insights is null